### ISSUE TITLE
Coding Style 

### ISSUE THREAD

**Issue by chungth** [hungtt57/khoaluan#3]
Laravel đang theo chuẩn PSR-2 và PSR-4
https://laravel.com/docs/5.2/contributions#coding-style
hiện tại project khá nhiều chỗ  indent , tên method, class chưa theo chuẩn này. 

Tham khảo thêm PSR-Fixer để có thể format và kiểm tra coding convention của dự án: 

http://cs.sensiolabs.org/

**Comment by hungtt57** [comment]
Vâng,đợt làm project này vẫn chưa có kinh nghiệm lên phong cách code vẫn hơi trẻ trâu.em đang khắc phục ạ
